apps/main/src/llamalend/features/market-list/LendingMarketsFilters.tsx

```diff
@@ -24,9 +23,22 @@ const Token = ({ symbol, data, field }: { symbol: string; data: LlamaMarket[]; f
     [data, field, symbol],
   )
 
-  return <TokenLabel blockchainId={chain} tooltip={symbol} address={address} label={symbol} size="lg" />
+  return <TokenLabel blockchainId={chain} tooltip={symbol} address={address} label={symbol} size="xl" />
 }
 
+const LendingMarketsFilterWrapper = ({ title, children }: { title?: string; children: React.ReactNode }) => (
```

Comment:
`Wrapper` doesn't add anything to the meaning of the component's name.
It relates to how it's used, not what it does. Anybody reading the code can directly see it wraps something because it has children :grin: 

Otherwise when we wanna wrap the wrapper we end up with `WrapperWrappers` and `WrapperWrapperWrappers` :grinning: 

```suggestion
const LendingMarketsFilter = ({ title, children }: { title?: string; children: ReactNode }) => (
```

Btw, we are trying to make the data table consistent between pages. Maybe we just call this `TableFilter` and move it to the `DataTable` folder?